__tests__/suitcss/valid.css

```diff
@@ -4,7 +4,7 @@
 .selector-1,
 .selector-2,
 .selector-3[type="text"] {
-  background: linear-gradient(#fff, rgba(0, 0, 0, 0.8));
+  background: linear-gradient(#fff, rgba(0 0 0 / 80%));
```

Comment:
Because we inherited `stylelint-standard`, it's applying a handful of rules that aren't being overridded by SUIT.

This one wants to use the modern space-separated syntax and percentage values for rgba. [Support for this](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgba()#space-separated_values) is everything but IE.

I can disable these two rules if you prefer.

Reply:
I'm having a way harsher reaction to converting the decimal alpha to a percentage for some reason

Reply:
Just pushed an update disabling that rule, while leaving the `color-function-notation` to prefer spaces. That means this is now valid: `rgba(0 0 0 / 0.8)`;